oppia

```diff
@@ -0,0 +1 @@
+Subproject commit e623bba4096afbce0aa6cbff65ca742fdb53e93d
```

Comment:
It's still here, and needs to be removed before this commit can be merged I'm afraid. Maybe "git rm"?